.gitignore

```diff
@@ -50,24 +50,18 @@ lib/ruby/gems
 lib/ruby/stdlib/*jopenssl*.jar
 lib/ruby/stdlib/bcpkix-jdk15on-*.jar
 lib/ruby/stdlib/bcprov-jdk15on-*.jar
-lib/ruby/stdlib/did_you_mean*
 lib/ruby/stdlib/gauntlet_rdoc.rb
 lib/ruby/stdlib/jar*
 lib/ruby/stdlib/jline
 lib/ruby/stdlib/jopenssl*
 lib/ruby/stdlib/krypt*
-lib/ruby/stdlib/net-telnet.rb
-lib/ruby/stdlib/net/telnet*
 lib/ruby/stdlib/openssl*
 lib/ruby/stdlib/org/
 lib/ruby/stdlib/ripper.jar
 lib/ruby/stdlib/rubygems/defaults/jruby_native.rb
 lib/ruby/stdlib/racc*
 lib/ruby/stdlib/rdoc*
-lib/ruby/stdlib/test*
 lib/ruby/stdlib/hoe*
-lib/ruby/stdlib/minitest*
-lib/ruby/stdlib/power_assert*
```

Comment:
don't we want to ignore all those whether they are default gems or auxiliary gems - they both will get installed during the build ?